package.json

```diff
@@ -9,20 +9,21 @@
     "bor": "cd test-blockchain && bash run-docker.sh",
     "bor:clean": "cd test-blockchain && bash stop-docker.sh",
     "truffle": "truffle",
-    "build": "truffle compile && node generate-artifacts.js && node flatten-contracts.js",
+    "build": "truffle compile && npm run artifacts && npm run flatten",
     "migrate:2": "truffle migrate --network root --f 2 --to 2",
     "migrate:3": "truffle migrate --network child --f 3 --to 3",
     "migrate:4": "truffle migrate --network root --f 4 --to 4",
     "migrate:5": "truffle migrate --network child --f 5 --to 5",
     "migrate": "npm run migrate:2 && npm run migrate:3 && npm run migrate:4 && npm run migrate:5",
+    "change-owners": "truffle migrate --f 6 --to 6",
```

Comment:
Shouldn't we use the script instead of migration? The truffle script could come in handy for the future as well.

Reply:
I started out with script then changed to migration since this action felt like a migration only. Changing back to script now :)